index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+  <link rel="stylesheet" type="text/css" href="css/index.css">
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="nav-center">
+      <div class="nav-header">
+        <img src="images/logo.svg" alt="High Landing">
+        <button type="button" class="btn nav-btn">
+          <i class="fa-solid fa-bars"></i>
+
+          </i>
+        </button>
+      </div>
+      <div class="nav-links">
+        <a href="#" class="nav-link list">About us</a>
+        <a href="#" class="nav-link list">How it works</a>
+        <a href="#" class="nav-link list">Pricing</a>
+        <a href="#" class="nav-link list">FAQs</a>
```

Comment:
@matycarolina `<a>` elements for navigation should be listed in a `<ul>` or `<ol>` list container